content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md

```diff
@@ -61,18 +61,17 @@ Suppose a Kubernetes cluster is running a device plugin that advertises resource
 on certain nodes. Here is an example of a pod requesting this resource to run a demo workload:
 
 ```yaml
----
 apiVersion: v1
 kind: Pod
 metadata:
   name: demo-pod
 spec:
   containers:
-    - name: demo-container-1
```

Comment:
@bmelbourne, thanks for the suggestion but as above comment suggested that it can be personal choice to have the indent or not, but indent does provide -> Clear hierarchy between parent and child elements.

Reply:
You can preview the content here and compare the content between [this file](https://deploy-preview-48385--kubernetes-io-main-staging.netlify.app/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins/) and [original file](https://kubernetes.io/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins/).